runhouse/servers/http/http_server.py

```diff
@@ -206,6 +211,7 @@ async def _add_username_to_span(request: Request, call_next):
             env_name=default_env_name,
             create=True,
             runtime_env=runtime_env,
+            logger=logger,
```

Comment:
just pass in the log level here? this would allow us to always import the same logger, instead of initializing different ones (related to other comment), and then the cluster servelt can just set the log level passed in: 
`logger.setLevel(log_level)`